flexget/utils/search.py

```diff
@@ -11,7 +11,16 @@ def clean_symbols(text):
     result = text
     if isinstance(result, unicode):
         result = normalize('NFKD', result)
-    return re.sub('[ \(\)\-_\[\]\.]+', ' ', result).lower()
+    result = re.sub('[ \(\)\-_\[\]\.]+', ' ', result).lower()
+    
+    """Special cases"""
+    result = re.sub("(\'s)+", 's', result)
+    result = re.sub("(\'t)+", 't', result)
+
+    """Leftovers"""
+    result = re.sub(r"[^a-zA-Z0-9 ]", "", result)
```

Comment:
This will remove all special chars like "à" "ã" and replace it by basic form "a" for example, replace  basic chars like "_" "-" by space (to accommodate words like Spider-Man and make it Spider Man). and in the end remove all special chars, making words like Marvel's to Marvel.